src/lib/blkalloc/varsize_blk_allocator.cpp

```diff
@@ -731,6 +731,17 @@ bool VarsizeBlkAllocator::is_blk_alloced(BlkId const& bid, bool use_lock) const
 }
 
 blk_num_t VarsizeBlkAllocator::available_blks() const { return get_total_blks() - get_used_blks(); }
+
+blk_num_t VarsizeBlkAllocator::get_freeable_nblks() const {
+    // TODO: implement this
```

Comment:
Can we put an assert here to indicate we are not supposed to arrive here? It can help unexpected error.
Same for fixed blk allocator.

Reply:
ACK